src/components/UTIL_PageMessageError.component

```diff
@@ -0,0 +1,25 @@
+<apex:component >
```

Comment:
I'm torn on this one. This component is a near exact duplicate of `UTIL_PageMessageInfo`. Rather than have these as exact duplicates we could do the following, which I think would make for a more re-usable component.

- Delete this new component
- Rename the `UTIL_PageMessageInfo` component to `UTIL_PageMessage` (it's only referenced in two places)
- Add an attribute to the component for severity that defaults to "info"
- Conditionally display the "info" or "error" label in the media__body block depending on the severity setting.

What do you think?
CC @RadSFDeveloper

Reply:
@force2b and @RadSFDeveloper  , There is already a component named UTIL_PageMessages and it is referenced in many classes. UTIL_PageMessage and UTIL_PageMessages are different names. If we rename the existing component, all references have to be changed as well.

Reply:
@force2b , I have renamed the component to UTIL_SingleToastMessage and updated the references to this component in the pages. Please check if it looks fine.